proto/desmos/posts/v1beta1/genesis.proto

```diff
@@ -20,6 +21,10 @@ message GenesisState {
   repeated desmos.posts.v1beta1.RegisteredReaction registered_reactions = 4
       [ (gogoproto.nullable) = false ];
   desmos.posts.v1beta1.Params params = 5 [ (gogoproto.nullable) = false ];
+  repeated desmos.posts.v1beta1.Report reports = 6 [
+    (gogoproto.moretags) = "yaml:\"reports\"",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
Can we move this before the params? I think making params the last object might make it more clean overll (I don't know why though lol)

Reply:
haha ok, we can have always params at the bottom.